samples/azure-quantum/resource-estimation/integer-factorization-with-cli/jobParamsBatching.json

```diff
@@ -0,0 +1,40 @@
+{
+    "items": [
```

Comment:
Not sure if it is a lot of work to change this word "items", but it doesnt really make sense in the relationship to whats in the array; could be more clear here

Reply:
The keyword `"items"` is to indicate that this is a batching job.